config.go

```diff
@@ -50,6 +51,7 @@ type Config struct {
 	TLSAuthorityCertificate     string              `yaml:"tls_authority_certificate"`
 	TLSCertificate              string              `yaml:"tls_certificate"`
 	TLSKey                      util.StringSecret   `yaml:"tls_key"`
+	Tls                         tls.Tls             `yaml:"tls"`
```

Comment:
What does the YAML representation of a `tls.Tls` look like?

Reply:
https://github.com/stripe/veneur/blob/master/util/tls/tls.go#L10-L14